cmd/minikube/cmd/start.go

```diff
@@ -171,32 +173,43 @@ func init() {
 
 // initClusterUpFlags creates the CLI flags which needs to be passed on to 'oc cluster up'
 func initStartFlags() {
-	startFlagSet.String(isoURL, constants.DefaultIsoUrl, "Location of the minishift iso")
-	startFlagSet.String(vmDriver, constants.DefaultVMDriver, fmt.Sprintf("VM driver is one of: %v", constants.SupportedVMDrivers))
-	startFlagSet.Int(memory, constants.DefaultMemory, "Amount of RAM allocated to the minishift VM")
-	startFlagSet.Int(cpus, constants.DefaultCPUS, "Number of CPUs allocated to the minishift VM")
-	startFlagSet.String(humanReadableDiskSize, constants.DefaultDiskSize, "Disk size allocated to the minishift VM (format: <number>[<unit>], where unit = b, k, m or g)")
+	startFlagSet.String(isoURL, constants.DefaultIsoUrl, "Location of the minishift ISO.")
+	//NEEDINFO: Is this the same as the binary? we should be consistent in how we refer to it. Also, we need to change the default from jdyson.
```

Comment:
btw, we should make clear that this is an URI, so either an http URL or file URI, see https://github.com/minishift/minishift/issues/182. Preferably the help text should have examples for both.